pkg/cmd/controller/cmd.go

```diff
@@ -10,11 +13,25 @@ import (
 	"github.com/openshift/service-ca-operator/pkg/version"
 )
 
+var (
+	featureGates map[string]bool
+)
+
 func NewController() *cobra.Command {
 	cmd := controllercmd.
-		NewControllerCommandConfig("service-ca-controller", version.Get(), controller.StartServiceCAControllers, clock.RealClock{}).
+		NewControllerCommandConfig("service-ca-controller", version.Get(), startServiceCAControllers, clock.RealClock{}).
 		NewCommand()
 	cmd.Use = "controller"
 	cmd.Short = "Start the Service CA controllers"
+	cmd.Flags().Var(utilflag.NewMapStringBool(&featureGates), "feature-gates", "Comma-separated list of key=value pairs that describe feature gates for alpha/experimental features.")
```

Comment:
Did you consider creating a component-base feature gate object to hold these? The component-base type includes an `AddFlag(*pflag.FlagSet)` method for this purpose, and it would behave exactly the same as the --feature-gates option on kube components.

Reply:
Seems OK in the context of fixing the bug.

> This might be a good idea for overall cmd flag standartization across the whole platform though

Having the operator wire them to a --feature-gates flag is our convention for at least OCM, KAS, KCM, and OAS. I think it's a good idea to stick to that in general. Making the mechanism consistent encourages the use of feature gates across the platform.